README.md

```diff
@@ -1,2 +1,20 @@
 # front-end
-The repo for the front-end sub-team
+Recycle Buddy Front End
+
+Recycle Buddy is an Mobile App designed during a Code Fellows Hack-a-thon between July 2018 and November 2018. The app is designed to take pictures of waste and turn it into meaningful recycling data for the user. 
+
+If you're interested in contributing to this project, please reach out to us through the GitHub repository. 
+
+# TODO: Include Installing, deploying, testing, etc. instructions
+
+
+# Version
+Currently in Pre-Alpha, Pre-Vertical Slice, Pre-MVP, Conceptual planning
+
+# Authors
+See CONTRIBUTING.md
```

Comment:
You could make this a link like this `[link name](../CONTRIBUTING.md)`

Reply:
thanks for letting me in on this knowledge